src/devtools/ui/Layout.tsx

```diff
@@ -1,10 +1,11 @@
 import { IconButton } from "@chakra-ui/button";
 import { Image } from "@chakra-ui/image";
 import { Heading, HStack, Stack } from "@chakra-ui/layout";
-import { ReactNode } from "react";
+import React, { ReactNode } from "react";
 import { MdSync } from "react-icons/md";
-import { requestRefresh, setOverrides } from "./controller";
-import logo from "./logo.svg";
+import { requestRefresh, setOverrides } from "../controller";
+// TODO Fix imports for SVGs
```

Comment:
Should we fix this before shipping? Looks like it would add a regression to the current devtools.

Reply:
Fixed